packages/replayio/package.json

```diff
@@ -4,7 +4,6 @@
   "description": "CLI tool for uploading and managing recordings",
   "bin": "./bin.js",
   "exports": {
-    ".": "./dist/index.js",
```

Comment:
do u plan to release v2 of this package? or do u consider this entrypoint to be somewhat internal and we'll skip that?

Reply:
I am prepared to do v2 but I could be convinced that no one is using this API and we can cheat semver a little.

Reply:
Do u remember why this was added? Who could be using this now?

Reply:
I don't think anyone is using it.